src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -14,12 +14,20 @@ namespace Microsoft.CodeAnalysis
 {
     public static class CSharpExtensions
     {
-        public static bool IsKind(this SyntaxToken token, SyntaxKind kind)
+        [Obsolete("Pass by reference instead.")]
+        public static bool IsKind(SyntaxToken token, SyntaxKind kind)
+            => token.IsKind(kind);
+
+        public static bool IsKind(this in SyntaxToken token, SyntaxKind kind)
```

Comment:
Is it possible to call this overload in reduced form? (i.e. `ref var token; token.IsKind();`)

Reply:
I was not able to find a way to call it without eliminating the other overload.